user/views.py

```diff
@@ -36,3 +38,54 @@ def form_valid(self, form):
             print(password)
             print(email)
             return HttpResponseForbidden()
+
+class ProfileView(ListView):
+    template_name = "user/profile.html"
+    model = Tweet
+
+    def get_queryset(self):
+        associated_id = self.kwargs['pk']
```

Comment:
関連するIDという命名では何に関連するか不明なので以下の変更を提案します。
意見があればコメントしてください。
```suggestion
        profile_user_id = self.kwargs['pk']
```

Reply:
命名を変更しました。